### PR TITLE
DCOS-48944 Use extension kid from npm

### PR DESCRIPTION
**Pull request by TattdCodeMonkey**

### CODE REVIEW

File: src/js/container.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { Container } from "inversify";
+import "reflect-metadata"; // Needed for inversify
```
Comment (pierrebeitz) [inline src/js/container.js]: Just out of curiosity: why wasn't that needed before?

**Comment by pierrebeitz** [comment]
If i get it correctly no component is using inversify anymore. Can we remove it from `package.json`?

**Comment by GeorgiSTodorov** [comment]
@pierrebeitz , if you think this is a blocking change, add a blocking review, otherwise someone might merge it as it has two approving reviews.

File: src/js/container.js (hunk 1 above)
Comment (nLight) [reply src/js/container.js]: We didn't use `extension-kid` it has been just sitting there. With this PR we do import the library thus engaging `@injectable`-like decorators.

**Comment by nLight** [comment]
@pierrebeitz we don't use `inversify` _directly_. Though it's a peer dep of `@extension-kid/core` https://github.com/dcos-labs/extension-kid/blob/master/package.json#L18

**Pull request merged**